suites/quincy/cephadm/tier-0.yaml

```diff
@@ -16,6 +16,8 @@ tests:
       desc: Install software pre-requisites for cluster deployment.
       module: install_prereq.py
       name: setup pre-requisites
+      config:
```

Comment:
@manasagowri I think you added this changes just for testing,can you revoke it back?
also can you please add test results to it.

Reply:
Done.